src/mono/mono/mini/method-to-ir.c

```diff
@@ -6078,6 +6078,14 @@ mono_method_to_ir (MonoCompile *cfg, MonoMethod *method, MonoBasicBlock *start_b
 		seq_points = FALSE;
 	}
 
+	if (method->wrapper_type == MONO_WRAPPER_OTHER)	{
+		WrapperInfo *info = mono_marshal_get_wrapper_info (method);
+		if (info->subtype == WRAPPER_SUBTYPE_INTERP_IN) {
+			/* We could hit a seq point before attaching to the JIT (#8338) */
+			seq_points = FALSE;
```

Comment:
@BrzVlad Are there other wrappers that might be used to go from native embedder code to the interp?

@vargaz Do we need sequence points in _any_ wrappers? How risky would it be to just disable them for all wrapper methods?  Are they used for stopping on a call to a pinvoke or something like that?

Reply:
that's the only one